### DISCUSSION TITLE
cut video incorrectly

### DISCUSSION THREAD

**Discussion by 42Frog** [WyattBlue/auto-editor#591] [4. Bugs] [answered]
Hi! I export to Adobe Premiere. Until recently, everything worked great. The program no longer cut the video correctly, makes large gaps and does not capture many sounds correctly. I reinstalled the program, tried different versions, videos, commands, i also
reinstalled Python, but the problem remains. Please help!

i use this command
`auto-editor "D:\video.mp4"  --edit audio:stream=1 --export premiere `

video info
`  - recommendedTimebase: 4983/100
 - video:
   - track 0:
     - codec: h264
     - fps: 2766650/55521
     - resolution: 1920x1080
     - aspect ratio: 16:9
     - pixel aspect ratio: 1
     - duration: 2220.84
     - pix fmt: yuv420p
     - timebase: 1/50000
     - bitrate: 39613237
     - lang: eng
 - audio:
   - track 0:
     - codec: aac
     - layout: stereo
     - samplerate: 48000
     - duration: 2220.7999791666666
     - bitrate: 196208
     - lang: eng
   - track 1:
     - codec: aac
     - layout: stereo
     - samplerate: 48000
     - duration: 2220.8855208333334
     - bitrate: 195278
     - lang: eng
 - container:
   - duration: 2220.885521
   - bitrate: 40017769`

![akMWUMh 1](https://github.com/user-attachments/assets/ded53700-58eb-4d95-b653-1d271d04eaa3)

![fGRcKw6 1](https://github.com/user-attachments/assets/dbfdd93d-13a2-43ab-99cc-431a31facb50)

**Comment by WyattBlue** [comment]
Can you run `git bisect`

**Comment by WyattBlue** [comment, marked as answer]
If you're also having this problem. Try using an input video that has a constant framerate (instead of a VFR). Doing that should fix cutting desyncs.
